setPL.sh

```diff
@@ -45,9 +45,8 @@ APP_NAME="setPL"
 #
 INTEL_MSR_PKG_POWER_LIMIT=0x610
 INTEL_PACKAGE_RAPL_LIMIT_0_0_0_MCHBAR_PCU=0x59a0
-INTEL_PL1_ENABLE_BITS_LOW=0x00008000
-INTEL_PL2_ENABLE_BITS_HIGH=0x00008000
-INTEL_PL1_PL2_ENABLE_BITS=$((INTEL_PL1_ENABLE_BITS_LOW | (INTEL_PL2_ENABLE_BITS_HIGH<<32)))
+INTEL_PL_ENABLE_BITS=0x00008000    #identical for PL1_ENABLE_BITS_LOW and PL2_ENABLE_BITS_HIGH
```

Comment:
The fact these two register halves incidental values for the desired field is incidental. They should still be kept separate for clarity and for future maintenance.